SportHub.UnitTests/UserListControllerTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AutoMapper;
+using Moq;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SportHub.API;
+using SportHub.Business;
+using SportHub.Business.Implementations;
+using SportHub.Data.Entities;
+using SportHub.Data.Interfaces;
+using SportHub.Controllers;
+using SportHub.Business.Interfaces;
+using Microsoft.AspNetCore.Http;
+
+namespace SportHub.UnitTests
+{
+
+    [TestFixture]
+    public class UsersControllerTests
+    {
+        private UserController _controller;
```

Comment:
Tests of controllers are integrations tests and should be in the IntegrationTests subproject

Reply:
Agree